### ISSUE TITLE
Missing Seaside 2nd edition DU-NL

### ISSUE THREAD

**Issue by merard4** [sumpfork/dominiontabs#444]
The cards from second edition of seaside are missing in Dutch. (I can provide names and descriptions of the cards if needed).

**Comment by sumpfork** [comment]
Yeah, they simply haven't been translated. Feel free to make a PR with the relevant JSON file changes -- I'm afraid that's the only way I have time to accept translations these days.

Given that the setup for this expansion upgrade is done, I think you mostly just need the card entries, see https://github.com/sumpfork/dominiontabs/pull/403/files#diff-c5956f14e6c1daf3f9135fbac96051cbfc7642c02cec198fb01a263858432c8b for reference.

**Comment by merard4** [comment]
Thanks for the feedback. I think I found the right JSON file. I translated everything and made a Pull Request. I've never done this before so I hope I didn't make any mistakes :)
